extension/src/test/resources/XliffCacheTest.da-DK copy.xlf

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Isn't there some better name for this available? 😉

Reply:
Zombie at the keyboard :zombie: